packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testResizeWithZoomBack.st

```diff
@@ -0,0 +1,10 @@
+zooming
+testResizeWithZoomBack
+	
+	| zoomLevel scrollPane zoomBack |
+	zoomLevel := 0.6.
+	zoomBack := 1.0.
+	scrollPane := SScrollPane forMorph: instance.
+	instance zoomTo: zoomLevel.
+	instance zoomTo: zoomBack.
+	self assert: (instance boundsInWorld area >= scrollPane boundsInWorld area).
```

Comment:
yes i used the bounds in world before. but because of rounding mistakes on float values it is not always asserted to work